WordPressVIPMinimum/Sniffs/UserExperience/AdminBarRemovalSniff.php

```diff
@@ -215,10 +213,8 @@ public function process_parameters( $stackPtr, $group_name, $matched_content, $p
 				}
 
 				$error = true;
-				if ( true === $this->remove_only && isset( $parameters[2]['raw'] ) ) {
-					if ( '__return_true' === $this->strip_quotes( $parameters[2]['raw'] ) ) {
-						$error = false;
-					}
+				if ( true === $this->remove_only && isset( $parameters[2]['raw'] ) && '__return_true' === $this->strip_quotes( $parameters[2]['raw'] ) ) {
```

Comment:
Should we remove the extra set of brackets enclosing `true === $this->remove_only && isset( $parameters[2]['raw']`?

Reply:
I don't think this observation is valid.